services/minespace-web/src/components/dashboard/mine/overview/WorkerInfoEmployee.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { Button, Col, Descriptions, Icon, Row, Tooltip, Typography } from "antd";
+import { fetchMineRecordById, updateMineRecord } from "@common/actionCreators/mineActionCreator";
+import { EDIT_PENCIL } from "@/constants/assets";
+import CustomPropTypes from "@/customPropTypes";
+import EditWorkerInformationForm from "@/components/Forms/mines/EditWorkerInformationForm";
+
+const { Title } = Typography;
+
+const propTypes = {
+  mine: PropTypes.objectOf(CustomPropTypes.mine).isRequired,
+  fetchMineRecordById: PropTypes.func.isRequired,
+  updateMineRecord: PropTypes.func.isRequired,
+};
+
+// eslint-disable-next-line react/prefer-stateless-function
+export class WorkerInfoEmployee extends Component {
```

Comment:
Is this disabl needed? This component has state so I'm surprised this error is being thrown in the linting.

Reply:
It was left when changed to class component. Thanks :)